### PR TITLE
WIP : feat : Added Homepage components

### PR DESCRIPTION
**Pull request by lannisterrr**
Added : Homepage and breaking into small components

### CODE REVIEW

File: src/pages/Home.js [hunk 1]
```diff
@@ -1,10 +1,63 @@
...
+          <h3 className="heading-2 t-c-3 f-bold p-h-4">Categories</h3>
+          <Link to="/listing" className="heading-3 t-c-2 f-bold p-h-4 see-all">
+            See All
+            <span>
+              <i className="fa fa-arrow-right p-h-3"></i>
+            </span>
+          </Link>
+        </header>
+        <main className="category-container">
+          <Category
```
Comment (GuptaEesh) [inline src/pages/Home.js:29]: Instead of writing <Category/> 5 times, you can make an array of objects and then loop here to put all details in 1 go.

**Review by GuptaEesh — COMMENTED** [review]
One change was mentioned. It will help reduce the redundant code.

File: src/pages/Home.js [hunk 2]
```diff
@@ -1,10 +1,63 @@
 import React from 'react';
```
Comment (junaidshaikh-js) [inline src/pages/Home.js:1]: from React V17.0 we don't need to import React.

**Review by junaidshaikh-js — APPROVED** [review]
Good Work! 

Changes suggested by @GuptaEesh can be done.

File: src/pages/Home.js (hunk 1 above)
Comment (lannisterrr) [reply src/pages/Home.js:29]: yes , will do at the last refactoring.

**Pull request merged**